app/models/user_track.rb

```diff
@@ -51,7 +70,7 @@ class DetermineAvailableExercisesIds
 
     # Get two datasets.
     # The first is the exercises with their counts of MATCHING preqreq concepts
```

Comment:
```suggestion
    # The first is the exercises with their counts of MATCHING prereq concepts
```

Reply:
Done in a separate commit on master. Thanks 👍